packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -187,11 +188,46 @@ describe('HTS Precompile Acceptance Tests', async function() {
   it('should be able to execute getApproved on nft', async function() {
     const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
 
-    const tx = await baseHTSContract.getApprovedPublic(NftHTSTokenContractAddress, NftSerialNumber, { gasLimit: 5_000_000 });
+      const tx = await baseHTSContract.getApprovedPublic(NftHTSTokenContractAddress, NftSerialNumber, { gasLimit: 5_000_000 });
     const { responseCode } = (await tx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args;
     expect(responseCode).to.equal(TX_SUCCESS_CODE);
 
     const { approved } = (await tx.wait()).events.filter(e => e.event === 'ApprovedAddress')[0].args;
     expect(approved).to.equal('0x0000000000000000000000000000000000000000');
   });
+
+  it('should be able to create a token with custom fees', async function() {
+    const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+    const tx = await baseHTSContract.createFungibleTokenWithCustomFeesPublic(accounts[0].wallet.address, HTSTokenContractAddress, {
+      value: ethers.BigNumber.from('20000000000000000000'),
+      gasLimit: 10_000_000
+    });
+    const txReceipt = await tx.wait();
+
+    const { responseCode } = txReceipt.events.filter(e => e.event === 'ResponseCode')[0].args;
+    expect(responseCode).to.equal(TX_SUCCESS_CODE);
+
+    const { tokenAddress } = txReceipt.events.filter(e => e.event === 'CreatedToken')[0].args;
+    expect(tokenAddress).to.be.not.null;
+
+    HTSTokenWithCustomFeesContractAddress = tokenAddress;
```

Comment:
Move this to the `beforeAll` if you want all tests to have access to this contract.
Or combine the 2 tests, create and get.